src/components/Category.jsx

```diff
@@ -0,0 +1,44 @@
+import StyledHeadline from '../components/StyledHeadline';
+import useStore from '../hooks/useStore';
+
+import StyledContainer from './StyledContainer';
+import VideoThumbnail from './VideoThumbnail';
+
+export default function Category() {
+	const videos = useStore(state => state.videos);
+
+	const allCategories = videos.map(video => {
+		return video.category;
+	});
+
+	let uniqueCategories = [...new Set(allCategories)];
```

Comment:
Why use `let` here? I would recommend to always use `const`.

Reply:
Good point, I'll change it

Reply:
> Good job, added some comments. Also, why do you sometimes use `.js` and sometimes `.jsx` for your files?

@macfrei I'm doing it for better readability. If an file has `.jsx` I know that it returns jsx. It will be compiled into a `.js` file in production.

Reply:
I would probably recommend to keep it consistent. Also, in detail, if you return a styled-component it basically returns a component with JSX 😉. In the end it is your decision, but usually the ending of a file is kept consistently within a project.